README.md

```diff
@@ -1,4 +1,5 @@
 <!-- markdownlint-disable-next-line -->
+
```

Comment:
```suggestion
```

Reply:
Hi Pradum, 
In my README.md, the line appears to be there. But I do remember deleting the line from the comments box that I was shown right before I completed the initial pull request. I'm not sure if that's what you're referring to. 

<img width="693" alt="Screen Shot 2022-06-14 at 10 14 50 AM" src="https://user-images.githubusercontent.com/75242911/173637421-0a46be43-00e1-4c3d-acc6-ed9a8dc3f526.png">

Reply:
If you running any linters or prettier, they add that.

You can click on commit suggestions to fix that :)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

Reply:
Oh okay. When I hover over the suggested change, I am not permitted to commit it due to not having write permissions. 
<img width="741" alt="Screen Shot 2022-06-14 at 10 28 07 AM" src="https://user-images.githubusercontent.com/75242911/173639832-d0c8b0e3-e894-4621-ab78-2a29fff126ae.png">

Reply:
oh sorry my bad, you have to remove the extra line.

That's weird. One this you can do is, directly commit those changes.

Reply:
Awesome. Thanks for the exposure and the guidance Pradum! :)